ppp_hal/requesthandler.py

```diff
@@ -40,22 +40,92 @@ def query(query, fields):
         mc.set(key, r, time=Config().memcached_timeout)
     return r
 
+PAPER_FIELDS = ('abstract_s', 'releasedDate_s', 'modifiedDateY_i',
+        'uri_s', 'halId_s', 'title_s', 'authFullName_s', 'arxivId_s',
+        'authFirstName_s', 'authLastName_s', 'authOrganism_s',
+        'labStructName_s', 'version_i', 'language_s')
+
+def graph_from_paper(paper):
+    same_as = list(filter(bool, (
+        paper['halId_s'],
+        paper.get('arxivId_s', None),
+        )))
+    organizations = paper.get('authOrganism_s', []) + \
+            paper.get('labStructName_s', [])
+    authors = [{'@context': 'http://schema.org/',
```

Comment:
No need of @context here. It is only useful for the root node of the graph serialization or if the context change.